api/src/main/java/com/messagebird/objects/ChildAccountCreateResponse.java

```diff
@@ -0,0 +1,51 @@
+package com.messagebird.objects;
+
+import java.util.List;
+
+public class ChildAccountCreateResponse {
```

Comment:
You can inherit from ChildAccountResponse instead of defining the `id` and the `name` properties. What do you think?